docs/core/setup/install-sdk.md

```diff
@@ -0,0 +1,160 @@
+---
+title: Install .NET Core SDK on Windows, Linux, and macOS
+description: Learn how to install .NET Core on Windows, Linux, and macOS. Discover the dependencies required to develop .NET Core apps.
+author: thraka
+ms.author: adegeo
+ms.date: 11/06/2019
+zone_pivot_groups: operating-systems-set-one
+---
+
+# Install the .NET Core SDK
+
+In this article you'll learn how to download and install the .NET Core SDK. The .NET Core SDK is used to create create .NET Core apps and libraries. The .NET Core runtime is always installed with the SDK.
+
+Here are links to the .NET Core downloads page by version:
+
+- [.NET Core 3.0 downloads](https://dotnet.microsoft.com/download/dotnet-core/3.0).
+- [.NET Core 2.2 downloads](https://dotnet.microsoft.com/download/dotnet-core/2.2).
+- [.NET Core 2.1 downloads](https://dotnet.microsoft.com/download/dotnet-core/2.1).
+
+## Install with an IDE
+
+When you install a supported IDE, you can also install .NET Core. By keeping the IDE up-to-date, you'll receive the latest supported .NET Core SDK.
+
+::: zone pivot="os-windows"
+
+- Download [Visual Studio](https://www.visualstudio.com/downloads/?utm_medium=microsoft&utm_source=docs.microsoft.com&utm_campaign=button+cta&utm_content=download+vs2019)\
+For more information, see [Visual Studio](#visual-studio).
+
+- Download [Visual Studio Code](https://code.visualstudio.com/)\
+For more information, see [Install with Visual Studio Code](#install-with-visual-studio-code).
+
+::: zone-end
+
+::: zone pivot="os-linux"
+
+- Download [Visual Studio Code](https://code.visualstudio.com/)\
+For more information, see [Install with Visual Studio Code](#install-with-visual-studio-code).
+
+::: zone-end
+
+::: zone pivot="os-macos"
+
+- Download [Visual Studio for Mac](https://visualstudio.microsoft.com/vs/mac/?utm_medium=microsoft&utm_source=docs.microsoft.com&utm_campaign=inline+link)\
+For more information, see [Install with Visual Studio for Mac](#install-with-visual-studio-for-mac).
+
+- Download [Visual Studio Code](https://code.visualstudio.com/)\
+For more information, see [Install with Visual Studio Code](#install-with-visual-studio-code).
+
+::: zone-end
+
+## Install with an installer
+
+Both Windows and macOS have standalone installers which can be used to install the .NET Core SDK.
+
+- Windows [x64 CPUs](https://dotnet.microsoft.com/download/thank-you/dotnet-sdk-3.0.100-windows-x64-installer) | [x32 CPUs](https://dotnet.microsoft.com/download/thank-you/dotnet-sdk-3.0.100-windows-x86-installer)
```

Comment:
these links get outdated fast. You already talked in the introduction where to find the downloads. One of them should go.